corehq/apps/userreports/templates/userreports/partials/report_filter_configuration.html

```diff
@@ -12,111 +12,123 @@
+                    <!-- ko if: $parent.optionsContainQuestions -->
+                    <input class="form-control" type="hidden" data-bind="
+                        questionsSelect: $parent.propertyOptions().concat(
+                            existsInCurrentVersion() ? [] : {value: dataSourceField(), label: ''}
+                        ),
+                        value: existsInCurrentVersion() ? property : dataSourceField,
+                        attr: {disabled: !existsInCurrentVersion()}
+                    "/>
+                    <!-- /ko -->
+                    <label class="help-block" data-bind="if: $parent.showWarnings() && !isNotBlank()">
+                        {% trans "Please select a property." %}</label>
+                    {# This is gross and I hate it #}
+                    <label class="help-block" style="white-space: pre-wrap;" data-bind="if: !existsInCurrentVersion()">
+                        {% blocktrans %}
+                            This property or question no longer exists in your app.
+                            You must delete this property to update this report.
+                        {% endblocktrans %}
+                    </label>
+                </td>
+                <td class="form-group">
+                    <input type="text" class="form-control" data-bind="
+                        value: displayText,
+                        attr: {disabled: !existsInCurrentVersion()},
+                        event: {keydown: function(){$data.userModifiedDisplayText = true; return true;}}
+                    ">
+                </td>
+                <!--ko if: $parent.hasFormatCol -->
+                <td>
+                    <select class="form-control" data-bind="
+                        options: $parent.formatOptions,
+                        value: format,
+                        attr: {disabled: !existsInCurrentVersion()}
+                    "></select>
+                </td>
+                <!--/ko-->
 
-        <td class="span1">
-            <i style="cursor: pointer;" title="Delete" data-bind="
-                click: function(){$parent.columns.remove($data)},
-                css: COMMCAREHQ.icons.DELETE
-            "></i>
-        </td>
-    {# there must be no whitespace between <tbody> and <tr> #}
-    {# the .hide().fadeIn() will fail badly on FireFox #}
-    </tr></tbody>
-    <tbody data-bind="visible: columns().length < 10" class="add-prop">
-        <tr>
-            <td></td>
-            <td colspan="4">
-                <button class="btn" data-bind="
-                    click: buttonHandler,
-                    text: buttonText
-                "></button>
-            </td>
-        </tr>
-    </tbody>
-</table>
+                <td class="span1">
+                    <i style="cursor: pointer;" title="Delete" data-bind="
+                        click: function(){$parent.columns.remove($data)},
+                        css: COMMCAREHQ.icons.DELETE + ' hq-icon-full'
```

Comment:
Ah I see. I agree `COMMCAREHQ.icons.DELETE` should be able to be used outside of a table cell.